src/styles/pk-spacing.scss

```diff
@@ -0,0 +1,13 @@
+//*****************************
+//*** PEACOCK SPACING SIZES ***
+//*****************************
+
+$spacing-xxs: 2px;
+$spacing-xs: 4px;
+$spacing-s: 8px;
+$spacing-m: 12px;
+$spacing-l: 16px;
+$spacing-xl: 20px;
+$spacing-xxl: 24px;
+$spacing-xxxl: 32px;
+$spacing-xxxxl: 64px;
```

Comment:
we now have more spacings... since they are on the "middle" i'd suggest to add them here

Reply:
I will update it. We probably need to change the naming. Any suggestion?